modules/core/include/opencv2/core/cuda.hpp

```diff
@@ -681,6 +684,9 @@ class CV_EXPORTS_W Stream
     //! returns true if stream object is not default (!= 0)
     operator bool_type() const;
 
+    //! return Pointer to CUDA stream
+    CV_WRAP void* cudaPtr() const;
```

Comment:
There is `StreamAccessor` to do it: https://docs.opencv.org/master/d6/df1/structcv_1_1cuda_1_1StreamAccessor.html. I think it's better to wrap accessor rather expose private fields.

Reply:
> StreamAccessor

Not sure that this can help to reach the final goal:

> to implement `__cuda_array_interface__` within the `GpuMat` python binding